### DISCUSSION TITLE
大家觉得最后会怎么解决这个问题呢

### DISCUSSION THREAD

**Discussion by cm-liang** [WeNeedHome/SummaryOfLoanSuspension#225] [General]
大家觉得最后会怎么解决这个问题呢

**Comment by Macabaka1** [comment]
都允许游行抗议的时候～ 说不定有转机？🐶

**Reply by DoDayum** [reply]
我看啊，转机就是等政府接盘，发现太多了，接不了，然后2007年的次贷再次发生，GG

**Reply by liuwenwu520** [reply]
爆发信任危机，金融危机才能解决吧，没钱是解决不了这个问题的

**Reply by lettermessenger** [reply]
什么？游行抗议？坦克车准备！！！！

**Reply by ghost** [reply]
赶紧游行，中共的坦克部队已经跃跃欲试了😁

**Reply by Liboce** [reply]
现在就想游行，不知道以后还想干什么

**Reply by jim486** [reply]
中小银行全部倒闭，留下几个大行银行取消纸币/强推数字货币

**Comment by wdhzfcc** [comment]
大闹大解决，小闹小解决，不闹不解决，闹了就解决

**Reply by Godwuqiang** [reply]
大闹大解决，小闹小解决，不闹不解决，闹了才解决

**Reply by YangTianKundream** [reply]
解决掉提出问题的人

**Reply by repairbean** [reply]
几百个白衬衫冲击抗议人群，忘啦？

**Reply by a6749920** [reply]
建议回家换套装备哈哈

**Comment by maydayzzzzz** [comment]
解决不了问题还解决不了你？

**Reply by TonyZZhang** [reply]
解决不了问题，就解决提出问题的人！

**Comment by Mikanzzc** [comment]
“记住，等会打人的时候把制服都脱了，看到敢拍照录像的那就往死里打”

**Reply by xiaoliutalk** [reply]
开门，查水表！

**Reply by GitHeP** [reply]
关卡：不要讲中文

**Reply by JerryPlayer123** [reply]
> 关卡：不要讲中文

Remember, no Chinese

**Comment by Vran9** [comment]
不了了之 ...

**Comment by suncunhu** [comment]


**Reply by simonDavid12138** [reply]
元凶是他们吗？

**Reply by macleer** [reply]
他们是代理人

**Reply by zifeiYv** [reply]
沈一石？

**Reply by smartleek** [reply]
只杀一个沈一石就拍手叫好的话，还会应某些人所需被推出无数个沈一石

**Reply by apang1992** [reply]
人家就是做生意的。要说拿最多的，还是炒房的人。开发商绝对没有炒房团赚的多。但凡可以破产，购房合同就自动取消了。现在政府不让破产，你让人家怎么办。

**Reply by royswale** [reply]
好一个阿Q，看完沙头，又要高呼万岁了

**Comment by jiang443** [comment]
有问题的楼盘多年前就有不少案例。按之前的情况，会慢慢一个个解决，但会拖延很久。

**Reply by superceII** [reply]
过去房价不断上涨，所以有接盘的。现在这情况，谁敢接

**Comment by 461807914** [comment]
1、取消预售制，取消公摊。
2、严格监管房地产市场和银行放贷规范。
3、给烂尾的业主赔偿，不纳入征信。
4、严惩违规开发商。

**Reply by wjiaqiao** [reply]
这每一条方案都很难执行到位

**Reply by daiyingwo** [reply]
赵老爷可能不大可能打赵家的家奴吧

**Reply by xiejun946** [reply]
怎么说尼，希望不要像唐山事件不了了之。作为每一个烂尾息息相关的业主，可怜的话不讲了，就是范恶心

**Reply by Mmmmeruem** [reply]
> 赵老爷可能不大可能打赵家的家奴吧

赵老爷是谁啊

**Reply by jiushi555** [reply]
每一条，明明是那么合理，但又那么像在做梦

**Comment by Kerrcy** [comment]
团结起来！！

**Reply by TheGreateJohny** [reply]
赵老爷们早就团结起来了。

**Comment by virusHF** [comment]
断臂求生，接触购房合同，同时接触贷款合同，扔掉首付，不再还贷款

**Comment by buwenzheng** [comment]
最好的结果当然是项目继续，大家正常偿还贷款。但是项目继续的话是需要有人来接盘的，原开发商是不可能了，新开发商基本也不会去接手，毕竟吃不到什么利润。gov也绝不可能自掏腰包。
猜测最终结果可能是，一直冷处理压热度，自媒体谁敢做相关内容就封号，这样如果压过去了也就算了。
如果发生了大面积抗议，我们gov的镇压手段大家都懂，维稳是第一要素。
无论怎么样可能最终都不会是个绝对有利于平民的结果

**Reply by PseudoCowboy** [reply]
压应该会压，但是钱咋办啊，就是不还贷

**Reply by CarberryChai** [reply]
这次不一样的，你想的有点多。要钱的好维稳，欠钱的怎么维稳，就是抄家砍头也还是没钱，就是不还你能咋滴？

**Reply by itsoursduty** [reply]
@CarberryChai  银行卡强制划款安排下，数字人名币一键充公了解下，安排你一个屁民还不是简简单单

**Comment by pacoblack** [comment]
直接从工资卡划扣

**Reply by xiejun946** [reply]
那就不上班了，到处借钱，刷信用卡，搞垮信用体系

**Reply by LeanFly** [reply]
扣工资太慢了，直接冻结名下所有存款、理财

**Comment by guyanyouyou** [comment]
拖……

**Comment by fiveZheng** [comment]
把提出问题的人解决了不就好了？

**Reply by redrain-wang** [reply]
是不是已经解决了提出问题 的 人？

**Comment by super-aviator** [comment]
印钱！

**Reply by zslavalon** [reply]
印到最后，1个馒头100块一个？

**Comment by mzk-cyber106808** [comment]
当然是全部上征信了  恶意不还贷🤡

**Comment by neroneroffy** [comment]
需要一次全面的金融危机来释放这些压力

**Comment by simonDavid12138** [comment]
如果闹得足够大,最终会是成立相关的专门的央企，对那些烂尾企业清算,清算完能自己盖起来的，自己盖,盖不起来的强制破产收购,当然这个收购了这些毒资产的央企肯定是资不抵债的,下一步就是国开行贷款给该央企,该央企获得国开行资金开始盖楼！然后该央企填补之前那一波倒闭的民企的市场空缺,成为中央直属的房企!至于贷款嘛,都是自家人,以后吃下全国一大份房地产开发的蛋糕慢慢还！这当中最难的就是注入资金那一步,那可是天量的资金,我们早上的煎饼果子又要涨价了！

**Reply by a6749920** [reply]
唐山也不是成立了专案组吗，就这点动静，抓了哪个大伞

**Comment by swzxsyh** [comment]
过段时间官方开始分化。对付中产阶级，承诺一批人，白制服打击一批人，历来如此。大部分人又不是一无所有，很容易被分化的。

**Reply by Plutus2Daniel** [reply]
已经出来方案了，估计机关事业单位接盘，然后打折销售给本单位员工

**Reply by simonDavid12138** [reply]
> 已经出来方案了，估计机关事业单位接盘，然后打折销售给本单位员工

在哪里看到的呀,那些烂尾楼已经是卖出去的呀,怎么销售,二次销售吗？

**Reply by PseudoCowboy** [reply]
> > 已经出来方案了，估计机关事业单位接盘，然后打折销售给本单位员工
> 
> 在哪里看到的呀,那些烂尾楼已经是卖出去的呀,怎么销售,二次销售吗？

卖出去了，你不还贷的话，不就收回去了吗，收回去再销售把

**Reply by lettermessenger** [reply]
斗地主！！！

**Reply by lettermessenger** [reply]
斗地主！！！

**Comment by hqqlucky** [comment]
拉一帮人，踩一帮人。夸一帮人，打一帮人。然后把一帮人当做自己人，天天发视频发文章。最后给点甜头给闹得凶的。实在不行，就把许老板这些人抓几个出来泄泄愤

**Comment by Mmmmeruem** [comment]
请客、斩首、收下当狗！

**Reply by suncunhu** [reply]
浦东就是上海 上海就是浦东

**Comment by lettermessenger** [comment]
参考河北烧烤打人，到底谁是黑势力的保护伞？ 事情闹的不够大？还是举报的人不够多？不还是就抓了几个小混混，就不了了之了？？？？

**Comment by a6749920** [comment]
至少搞点钱放手上才安全点，不过中国讨论问题一直在外国的网站上的确惹得外国人奇怪哈哈

**Comment by wellzm** [comment]
印钱。一方面国企 比如城投 来接,  继续施工.  另一方面， 苦的是业主,  交付的房子质量可能偷工减料,  甚至可能重新规划小区设计,  比如景观什么可能被取消 。 部分业主为了搬进去不得不妥协.

**Comment by mehhss** [comment]
抓进新疆集中营进行劳改

**Reply by Jenghan** [reply]
全部抓起來豈不更還不起貸款 (而且這段時間疫情防控已經證明，鄰國無處不新疆)

就算銀行門戶大開，強迫送每個受災戶兩百萬人民幣無息貸款，讓大家開心把樓房都完成... 
顯然也只會堆積更大的問題。

**Reply by ppinkiris** [reply]
傻逼

**Comment by blisee** [comment]
会不会像之前的匹凸匹

**Comment by CodeJiang** [comment]
银监会不是针对断贷做了指示吗？ 要银行承担起社会责任，看相关银行怎么接招了

**Comment by Jenghan** [comment]
這問題是二三十年來在貴國所有人眼皮下逐漸長起來的。
整個系統是數以億計的人 (甚至包含海外投資者)、長時間以來由小到大，一塊塊骨牌銜接堆砌起來的。 累積了龐大的崩塌能量... 

有操弄 (只能發布正面消息)、貪婪 (所有參與者都追逐獲利)、惡意 (撈了錢就跑)、怯懦 (不問是非、只求自保) 
全世界第一人口大國、第二大經濟體，不知有多少比例是用偉大光明正確 + 鋼筋水泥堆堆砌而成。

如果有人能在短時間內用甚麼妙招化解，消累世惡業 (所有人投入的時間、精力、財力物力加起來) 於無形...  那真的是要翻新物理定律了。

**Comment by jiushi555** [comment]
最后会怎么解决？
西安孕妇、上海被强行带走的一家人、上海死亡的护士、上海心脏病（还是脑梗）猝死的父亲、强行打疫苗死亡的老师、丹东老人、唐山烧烤摊......最后解决了？
实在不行了，参考河南：打一批、抓一批、安抚一批，齐活

**Comment by toxin08x** [comment]
https://www.youtube.com/watch?v=YsIoSCuTHqk&t=1s

这个视频提及了可能的3个处理方案

**Comment by a6749920** [comment]
河南的办法肯定会借鉴一下，减小一点矛盾是一点，然后跟你耗时间拖，有可能不还贷工作都没有

**Reply by guoyongchang** [reply]
肯定的，比如社保登记要求非黑户。退一步讲哪怕不硬着来，软着偷偷的来也行。

现在不是已经有单位要求签署不断贷承诺书吗，不然就不给升职还是续签劳动合同，再结合有的单位要求历史无阳。

到最后就是GOV没明面要求，各大企业把事办了。真有民众揪着公司不放，那GOV下场让出头的公司自罚三杯另许诺些福利即可。

到时候政府的目的达到了
人民的诉求（惩罚`自作主张`的公司）达到了
公司的福利也得到了

三赢，意思是中国赢三次，**体面！**

**Comment by yazhou-zyz** [comment]
政府全权接管，回到共产时代

**Comment by ckckckckk** [comment]
“房子我不要了，别打了！”

**Reply by jiushi555** [reply]
不要了？不再买两套能让你走？

**Comment by Streaking-snail** [comment]
对于有产阶级：不予理会，反正他们舍不得那点瓶瓶罐罐，不会闹事，当无事发生就行
对于无产阶级：先打压，捂嘴。在这个基础上放弃的就等人死债消，坚持的就小闹小解决，大闹大解决。
都是老祖宗传下来的精华

**Comment by pengyidong** [comment]
分而治之

**Comment by rechardlc** [comment]
没啥用~在特色下，基本最后都是不了了之~然后继续奏乐~

**Comment by puroyakyu** [comment]
让所有业主额外再凑一笔钱出来找建筑把楼盘完成，这笔钱不会很大，因为楼盘可能只差两、三成就完工，用成本价分摊开可能每户几万到十几万。如果业主没钱可以跟再以优惠利率跟银行借。原来的按揭也会享受利息优惠。反正韭菜就是要割的暖心。

**Comment by selphy65** [comment]
估计也是分而治之，不了了之，然后对一些顽固分子，抹了

**Comment by miaodi** [comment]
认栽呗，不行报复下社会。

**Comment by ppinkiris** [comment]
政府兜底一套一套交呗
不然像河南这种地方以后狗都不买

**Reply by haoww** [reply]
跑不了房贷该交还得交

**Comment by lvzhengkun** [comment]
“只要不出乱子就慢慢拖，上面没给意见我们就不能动，要学会把问题留给后人，也要相信后人的智慧，我们要做的就是维稳，不利于团结的话不能说，不利于团结的事不能做，各个部门都要以稳定为头等大事，我丑话撂在这，哪个部门治下出了事死了人上了新闻，部门一把手就主动向市委辞职吧。”

**Reply by yjeff112** [reply]
这才是权贵阶层们的真实想法，一切都不要耽误自己捞钱。还有人妄想着不讨论政治给老爷们跪下老爷们就会给赏口饭吃，老爷们只会当你们是一群听话的狗，要做的就是乖乖听话维持现有的社会秩序继续让他们捞钱，然后鼓励再多生点让后代继续给他们后代当牛做马

**Comment by aiheihe** [comment]
大概率不了了之 
这个事情一点热度都没有了
如果真的会大面积停贷可能还有点用
但像现在这样喊一喊是没有用的
有益于人民的政策是要用流血逼出来的

**Comment by Siryedu** [comment]
政府政策性定向投放房地产金融政策贷款，房地产开发商贷款盖房，烂尾楼业主加钱补贴开发商亏损，最后的处理就是业主掏钱补差价、全民承担放水通胀。

**Comment by a6749920** [comment]
房地产现在都可以和教育 公务员的业绩挂钩了，挂钩其他的还会晚吗

**Reply by frshgrass** [reply]
其实怎么说呢 已经在历史的关口了 有的人还不自知 腆着脸赖着

韩国和台湾都是在经历了经济高速发展 贫富差距加大后 开始由原来的威权体制转向民主 韩国有光州事件 台湾有美丽岛事件 


可以中国大陆也在这个进程中 而且基本可以确定就是房地产垮掉后 全民财富蒸发 大量失业后 人民开始转向上街 

经济规律是不会消失的 可以人为推迟 但是会以更加迅猛败坏的方式到来（美国上世纪禁酒令，结果反而更多人酗酒，而且真正的做大美国黑帮）


所以泡沫一定是会破的 只不过财富蒸发后 大面积失业后 这股社会不满的情绪要怎么发泄：

  1，推翻政府，建立民主（同为儒家文化圈，同为威权体制时期经历经济高速发展，极大可能和韩台一样，经历民主运动后开始民主选举）

  2，借由这些年培养的小粉红学生，文革回潮。但是这是相当恐怖的，没有人有足够的权威一定能独善其身，因为能广泛打出来的就是毛的旗号，借毛是权威，各派血斗。有人说自己是领袖那只是他自以为是，他没有这样的权威（文革时期只有毛作为开国皇帝是安全的，现在文革谁也说不好谁安全。各种泡打司令部，各种红旗打红旗，就像伊斯兰各种教派血斗）但是实话说现在的小粉红品客 还不够多 这种概率比较小 

也终于快到了 解下束缚在民族上的枷锁，之后 大家可以松一口气 繁衍生息 。

**Comment by a6749920** [comment]
暴风雨的前面都是比较平静的

**Comment by hclgdb** [comment]
拖着就行了。拖到他们闹不动自认倒霉的时候就好了。参考西安烂尾十年不也没咋着吗？哈哈

**Comment by redrain-wang** [comment]
您好，邮件已经收到，我会尽快查收的！

**Comment by xijinping0** [comment]
最后怎么解决？现在各地财政越来越吃紧，哪来钱填烂尾楼这些窟窿。二十大之后，再闹的直接拉清单！

[![拉清单](https://img.youtube.com/vi/iBg0UhhV72E/0.jpg)](https://youtu.be/iBg0UhhV72E?t=113)

https://github.com/xijinping0
《[习近平与他的情人们](https://lovers.xijinping.one)》

**Comment by b4o** [comment]
跟香港模式差不多，这事必须倒闭几个大企业，必须有人付出生命，必须让那些既得利益者把事情弄到不可收拾，然后才有理由机会出重拳根治，，，中国人太讲人情了，反而把法放到了下面

**Reply by Cegazhos** [reply]
好的情况是央企国企兜底借钱给施工方出来建完交房。坏的情况 是根本不管， 只负责镇压闹事的。

**Comment by redrain-wang** [comment]
您好，邮件已经收到，我会尽快查收的！
